### DISCUSSION TITLE
White mesh and sulcal .arg mismatched - transformation problem?

### DISCUSSION THREAD

**Discussion by HdeV999** [brainvisa/brainvisa.github.io#136] [Q&A] [answered]
Hello, 
The morphologist pipeline has been applied on a series of subjects. Occasionally, a subject's white matter mesh and sulcal .arg are mismatched in terms of transformations/referential when opening them with 'inspect multiple sulci graph'. 
I checked the transformations of both files using the data browser, and they differ: specifically, the transformations of the white mesh is suspicious, with mostly 0 and 1 values when the transformations of the sulcal graph are multi-decimal numerics. 
I should probably manually change the transformations of the white mesh by copying the one of the .arg, how can I do that?
Many thanks!

**Comment by denisri** [comment]
Hi,
Does it happen systematically ? What are the "transformation files" ? You mean the .trm file which goes to Talairach or MNI space (there should be only 1 for a given raw T1 image, the graph and the white mesh should use the same in brainvisa), or the white image header information ? Do they have a different referential in anatomist when you display them ?
Have you tried to use the "check database" process in brainvisa (data management toolbox) ?

**Reply by HdeV999** [reply]
Hi, sorry for the very slow reply. No, this doesn't happen systematically, maybe just one subject in a database of ~600 subjects. 
When I drag and drop the .arg and the Lwhite of the subject in a data browser window using anatomist, I get different vectors in the 'transformations' field, so I guess that's part of the header information?
Yup, I just ran the "check database" process, and it didn't solve the issue... 
The two objects are in the 'referential of raw t1 MRI' but then in the browser in the 'referentials' field, the Lwhite says ['Coordinates aligned to another file or to anatomical truth',  'Coordinates aligned to another file or to anatomical truth'] while the .arg says ['Scanner-based anatomical coordinates', 'Talairach-AC/PC-Anatomist'], I don't know if that's relevant.

**Reply by HdeV999** [reply, marked as answer]
Hi again. Actually, reprocessing the subject solved the issue. Thanks!

**Reply by denisri** [reply]
OK, good news...
